src/main/java/teammates/ui/template/FeedbackTemplateQuestionInfo.java

```diff
@@ -0,0 +1,140 @@
+        }
+
+        public Builder withFeedbackQuestionType(String questionType) {
+            if (questionType != null) {
+                feedbackTemplateQuestionInfo.qnType = questionType;
+            }
+            return this;
+        }
+
+        public Builder withFeedbackQuestionText(String questionText) {
+            if (questionText != null) {
+                feedbackTemplateQuestionInfo.qnText = questionText;
+            }
+            return this;
+        }
+
+        public Builder withFeedbackQuestionFeedbackPath(String feedbackQuestionFeedbackPath) {
+            if (feedbackQuestionFeedbackPath != null) {
+                feedbackTemplateQuestionInfo.qnFeedbackPath = feedbackQuestionFeedbackPath;
+            }
+            return this;
+        }
+
+        public Builder withFeedbackQuestionVisibilityOption(String feedbackQuestionVisibilityOption) {
+            if (feedbackQuestionVisibilityOption != null) {
+                feedbackTemplateQuestionInfo.qnVisibilityOption = feedbackQuestionVisibilityOption;
+            }
+            return this;
+        }
+
+        public Builder withFeedbackQuestionVisibilityHints(List<String> questionVisibilityHints) {
+            feedbackTemplateQuestionInfo.qnVisibilityHints =
+                    questionVisibilityHints == null ? new ArrayList<>()
+                            : new ArrayList<>(questionVisibilityHints);
+            return this;
+        }
+
+        public FeedbackTemplateQuestionInfo build() {
+            return feedbackTemplateQuestionInfo;
+        }
+    }
+
+    public int getQnNumber() {
+        return qnNumber;
+    }
+
+    public String getQnType() {
+        return qnType;
+    }
+
+    public String getQnText() {
+        return qnText;
+    }
+
+    public String getQnFeedbackPath() {
+        return qnFeedbackPath;
+    }
+
+    public String getQnVisibilityOption() {
+        return qnVisibilityOption;
+    }
+
+    public List<String> getQnVisibilityHints() {
+        return qnVisibilityHints;
+    }
+
+    public String getQnDescription() {
+        switch (qnNumber) {
+        case 1:
+            return "Use <b>peer estimates</b> to determine the <b>work distribution percentage</b> "
+                    + "among team members in <b>a team activity</b>";
+        case 2:
+            return "Ask each student to describe something related to themselves i.e., <b>self-reflection</b>";
+        case 3:
+            return "Ask each student to give <b>confidential peer feedback (qualitative)</b> to other <b>team members</b>";
+        case 4:
+            return "Ask each student to give <b>confidential feedback (qualitative)</b> about the team behaviour";
+        case 5:
+            return "Ask each student to give <b>comments about other team members, in confidence</b>";
+        default:
+            return null;
+        }
+    }
+
+    public String getQuestionSpecificEditFormHtml() {
+
+        if (qnNumber == 1) {
```

Comment:
Add a comment to explain that question 1 is TeamContribution and rest are Essay Questions. Will be necessary if we decide to add more Template questions